draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -1578,6 +1578,26 @@ Procedure:
 5. return W
 ```
 
+# Privacy Considerations
+
+This section will go through threats to the Prover's privacy. Note that a BBS proof is unlinkable both against Verifiers coalition and Verifiers coalition with the Issuer. The following sections will describe possible threats, resulting from side chanel information or identifying disclosed messages, that could compromise the unlinkability property of the BBS proof. Such threats, if exploited, could lead to correlation of the Prover's interactions with different Verifiers, resulting to fingerprinting attacks on the Prover's activity.
```

Comment:
Is the typical term in this context coalition or correlation?

Reply:
AFAIK yes. Correlation generally refers to breaking the unlinkability property of the proofs (i.e., "correlating" multiple proofs presenting by the same Prover together)

Coalition refers to multiple parties (Verifiers and potentially the Issuer) working together to track the user's activity (the proofs they produce).